dhctl/pkg/kubernetes/actions/resources/resources.go

```diff
@@ -98,21 +98,33 @@ func (c *Creator) createAll() error {
 
 		for i, resource := range c.resources {
 			if _, ok := addedResourcesIndexes[i]; !ok {
+				log.DebugF("Remain resource %s\n", resource.String())
 				remainResources = append(remainResources, resource)
 			}
 		}
 
 		c.resources = remainResources
+		log.DebugF("Remain resources: %d\n", len(c.resources))
 	}()
 
-	if err := c.ensureRequiredNamespacesExist(); err != nil {
+	log.DebugLn("start ensureRequiredNamespacesExist")
+
+	resourcesToSkipInCurrentIteration, err := c.ensureRequiredNamespacesExist()
```

Comment:
It took me a while to figure out that there is a connectiong between this `resourcesToSkipInCurrentIteration` and `c.resources` and they are linked via indexes. There should be at least a comment about that&

Reply:
done